core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowApplication.scala

```diff
@@ -109,6 +109,10 @@ object CloudflowApplication {
       applicationSpec
     )
 
+  def getOwnerReferences(app: CR) = List(
+    OwnerReference(app.apiVersion, app.kind, app.metadata.name, app.metadata.uid, Some(true), Some(true))
```

Comment:
Some true is not the whole truth, right? ;-)

Reply:
hehe, questionable DSL indeed.